pyproject.toml

```diff
@@ -14,4 +14,4 @@ dynamic = ["version", "description"]
 Home = "https://github.com/evalf/nutils-SI/"
 
 [tool.flit.module]
-name = "nutils.SI"
```

Comment:
You can actually remove the `[tool.flit.module]` section. It's derived from `project.name`.